view/scn-social-auth/user/login.phtml

```diff
@@ -1,10 +1,6 @@
 <?php
-$socialSignIn = true;
+echo '<h1>' . $this->translate('Social Sign In') . '</h1>';
```

Comment:
It would break for users of this module that might depend on it not printing "Social Sign In" under certain conditions.